### PR TITLE
Feature/add client repository

### PR DESCRIPTION
**Pull request by Maria-Zurakowska**
1. The code consisted of business entity classes as well as CourseRepository interface with an implementing class (ListBasedCourseOperator)
2. The project required additional repository - ClientRepository to handle client management
3. I've added a new interface - ClientRepository with typical CRUD methods to handle client management:
createClient(String name, String lastName), printAllClients(), findClientsByLastName(String lastName), updateClient(Client client, String name, String lastName) and deleteClient(Client client)
and their implementation | I've also added ClientOperator methods to School class for it to be responsible for governing Clients

### CODE REVIEW

File: src/main/java/com/koreanlanguageschool/KoreanLanguageSchoolApplication.java [hunk 1]
```diff
@@ -36,4 +35,13 @@ private static void getCourseClients() {
 
 		System.out.println(clientsByCourse);
 	}
+
+	private static void createClient(){
```
Comment (PaniOdProgramowania) [inline src/main/java/com/koreanlanguageschool/KoreanLanguageSchoolApplication.java]: let's rename this method - the name `create` indicates something would be returned but it's actually `void`... so I think a better name would be even `test`. Or, maybe even put the inside of this method into the `main` method and test the `operator` there :)

**Review by PaniOdProgramowania — COMMENTED** [review]
so far it looks good :) I added just one comment. Please apply changes and then let me know once this is ready for re-reviewing:)

File: src/main/java/com/koreanlanguageschool/KoreanLanguageSchoolApplication.java (hunk 1 above)
Comment (Maria-Zurakowska) [reply src/main/java/com/koreanlanguageschool/KoreanLanguageSchoolApplication.java]: Ok, done

**Comment by Maria-Zurakowska** [comment]
I needed to create a School object with all of its parameters in order to later invoke the printAllClients() method from the said class

**Review by PaniOdProgramowania — APPROVED** [review]
great! feel free to merge 🎖️

**Comment by Maria-Zurakowska** [comment]
Ready to merge

**Pull request merged**